plugins/tpm/fu-plugin-tpm.c

```diff
@@ -141,6 +141,13 @@ fu_plugin_tpm_add_security_attr_eventlog(FuPlugin *plugin, FuSecurityAttrs *attr
 	g_autoptr(GPtrArray) pcr0s_calc = NULL;
 	g_autoptr(GPtrArray) pcr0s_real = NULL;
 
+	/* check reconstructed to PCR0 */
+	if (fu_plugin_has_flag(plugin, FWUPD_PLUGIN_FLAG_DISABLED) || data->bios_device == NULL ||
+	    data->ev_items == NULL) {
+		fwupd_security_attr_set_result(attr, FWUPD_SECURITY_ATTR_RESULT_NOT_FOUND);
```

Comment:
it needs to be a little more subtle how this is done - the attribute needs to be made first and then the guids only added if `data->tpm_device` is not `NULL`

Reply:
I should add a pre-push hook to run the tests… Now fixed.

Reply:
> I should add a pre-push hook to run the tests… Now fixed.

That's a good idea.  If you want to add something to `contrib/setup` it would be a good thing to set everyone up for success and higher quality code.